webapp/README.md

```diff
@@ -112,6 +112,22 @@ Re-factoring RSU manager to utilize Redux Toolkit for state management
    - References
      - WzdxMap.js - read and load WZDx data
 
+## Feature Flags
+
+This application has the ability to disable certain features based on environment variables. For each of these variables, the feature will be enabled if the variable is anything but 'false'. These features include:

```

Comment:
It might be worth noting a disclaimer that the corresponding feature API endpoints would still be exposed. If this is planned to be changed in the near future then we can leave this as is.

Reply:
I added flags to the API, and noted that in the main README